### PR TITLE
Remove Tag Default Color

### LINKED ISSUE #331: Remove Default Color
image

- Ensure that no Tag relies on this default behavior and that the color is explicitly defined when being called

### PR DESCRIPTION
**Pull request by Lackline**
remove tag default color

### CODE REVIEW

File: src/components/Admin/Tag.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-const Tag = ({ color = "gray", text, onClick = null, classes }) => {
+const Tag = ({ color, text, onClick = null, classes }) => {
   return (
     <div
       className={`${classes} ${colors[color].background} ${
         colors[color].text
       } ${
         onClick && `hover:cursor-pointer ${colors[color].hover}`
-      } whitespace-nowrap px-2 py-0.5 rounded text-xs md:text-sm w-fit m-0`}
+      } whitespace-nowrap px-2 py-0.5 rounded-md text-xs md:text-sm w-fit m-0`}
```
Comment (shahdivyank) [inline src/components/Admin/Tag.jsx]: dont need this

File: src/components/Admin/Tag.jsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import { colors } from "@/data/TagColors";
 
-const Tag = ({ color = "gray", text, onClick = null, classes }) => {
```
Comment (shahdivyank) [inline src/components/Admin/Tag.jsx:3]: make sure that all the tags r working fine despite this, many places depended on the default color

Comment (shahdivyank) [reply src/components/Admin/Tag.jsx:3]: remove the `= null` as well

**Comment by shahdivyank** [comment]
attach issue

File: src/components/Admin/Tag.jsx (hunk 2 above)
Comment (Lackline) [reply src/components/Admin/Tag.jsx:3]: I am not sure what will be a working tag should be function as, but so far I checked that tags across working main pages seems to interact normally.

**Review by shahdivyank — APPROVED** [review]

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**